examples/quickstart/customize_async.cpp

```diff
@@ -79,5 +79,10 @@ int main()
         f.wait();
     }
 
-    return 0;
+    return hpx::local::finalize();
+}
+
+int main(int argc, char* argv[])
```

Comment:
Unrelated to this PR: should we add support for `#include <hpx/local/hpx_main.hpp>`?

Reply:
My preference is not to add it.

Reply:
What's your rationale for this? Wouldn't it simplify porting codes?

Reply:
Not noticeably. While it's convenient on the surface, the implementation is complex, error prone and not really portable. That's of course only my opinion and I can see how others might think that's a fine tradeoff.